contracts/cw20-gov/src/contract.rs

```diff
@@ -80,8 +80,8 @@ fn verify_png_logo(logo: &[u8]) -> Result<(), ContractError> {
 /// Checks if passed logo is correct, and if not, returns an error
 fn verify_logo(logo: &Logo) -> Result<(), ContractError> {
     match logo {
-        Logo::Embedded(EmbeddedLogo::Svg(logo)) => verify_xml_logo(&logo),
-        Logo::Embedded(EmbeddedLogo::Png(logo)) => verify_png_logo(&logo),
+        Logo::Embedded(EmbeddedLogo::Svg(logo)) => verify_xml_logo(logo),
+        Logo::Embedded(EmbeddedLogo::Png(logo)) => verify_png_logo(logo),
```

Comment:
Does this change need to be pushed upstream? Seems incorrect to need a reference of a reference.

Reply:
This was clippy complaint. :shrug:

Clippy is :crab: :crown:.

Reply:
the one true king

Reply:
Praise be. :crab: :crown: